src/containers/shared/utils.js

```diff
@@ -135,11 +147,12 @@ export function formatPrice(number, lang = 'en-US', currency = 'USD', decimals =
 
 // Document: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat
 export const localizeDate = (date, lang = 'en-US', options = {}) => {
-  // TODO: default config
+  const config = options ?? DATE_DEFAULT_OPTIONS;
```

Comment:
```suggestion
  const config = {
  ...DATE_DEFAULT_OPTIONS,
  options
};
```
This way, users can specify just part of the list instead of needing to repeat the default options for everything if they only want to display one part differently.

Reply:
I originally had it that way but then found `dateStyle` didnt work if the options were extended.  I need to think through some of this some more